CONTRIBUTING.md

```diff
@@ -44,11 +44,9 @@ This is a rough outline of what a contributor's workflow looks like:
 
 Thanks for your contributions!
 
-### Coding Style
+### Code Style
 
-CoreOS projects written in Go follow a set of style guidelines that we've documented
-[here](https://github.com/coreos/docs/tree/master/golang). Please follow them when
-working on your contributions.
+Code should be formatted according to the output of [rustfmt](https://github.com/rust-lang-nursery/rustfmt)
```

Comment:
Can we add a `rustfmt` call to the Travis job similar to how we run `gofmt` on other projects?

Reply:
I'm going to add this in the future to both coreos-metadata and update-ssh-keys.